extensions/wikia/FacebookPreferences/tests/FacebookPreferencesModuleServiceTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use Swagger\Client\ApiException;
+use Swagger\Client\ExternalAuth\Api\FacebookApi;
+use Swagger\Client\ExternalAuth\Models\LinkedFacebookAccount;
+
+class FacebookPreferencesModuleServiceTest extends TestCase {
+	/** @var int $userId */
+	private $userId = 1563;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$userId" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1068)